### PR TITLE
Let the custom title bar support windows11 quick layout

### PR DESCRIPTION
**Pull request by qianniancn**
Use DWM to disable the title bar, so that the window has the default window style, support windows 11 fast layout arrangement

https://user-images.githubusercontent.com/5478237/210163060-46233da4-dba0-46fd-84db-5f94283f4b9c.mp4

### CODE REVIEW

**Comment by gedw99** [comment]
that is cool. Nice work.

File: app/os_windows.go [hunk 1]
```diff
@@ -662,9 +691,9 @@ func (w *window) Configure(options []Option) {
 	swpStyle := uintptr(windows.SWP_NOZORDER | windows.SWP_FRAMECHANGED)
 	winStyle := uintptr(windows.WS_OVERLAPPEDWINDOW)
 	style &^= winStyle
-	if !w.config.Decorated {
-		winStyle = 0
-	}
+	//if !w.config.Decorated {
+	//	winStyle = 0
+	//}
```
Comment (eliasnaur) [inline app/os_windows.go:696]: Delete. Judging from your video, I assume switching between undecorated/decorated still works?

Comment (qianniancn) [reply app/os_windows.go:696]: Yes, it still works. Now the WM_NCCALCSIZE message controls how to show or hide the title bar.

Comment (qianniancn) [reply app/os_windows.go:696]: See: https://learn.microsoft.com/en-us/windows/win32/dwm/customframe

Comment (eliasnaur) [reply app/os_windows.go:696]: Thanks. Please delete the disabled lines completely.

**Comment by qianniancn** [comment]
I found a border problem in windows7 and haven't found a solution yet.

**Comment by whereswaldon** [comment]
@qianniancn Did you choose to close this because the problem on Windows 7 is still unsolved?

**Pull request closed without merging**